tests/sentry/workflow_engine/endpoints/test_organization_detector_details.py

```diff
@@ -300,7 +300,12 @@ def test_update_owner_to_user(self) -> None:
         assert detector.owner.identifier == self.user.get_actor_identifier()
 
         # Verify serialized response includes owner
-        assert response.data["owner"] == self.user.get_actor_identifier()
+        assert response.data["owner"] == {
```

Comment:
nit: I probably would have just used the actor serializer here. Since we don't really want to test the serialization format of the actor. Testing like this turns it more into an Integration style test.